### ISSUE TITLE
Netedit creates minor uncontrolled connections in a TL intersection

### ISSUE THREAD

**Issue by m-kro** [eclipse/sumo#5286]
Possibly related to #5021 . Happens at an intersection where some connections are not controlled by the traffic light. Netedit automatically sets some uncontrolled connections to 'minor' (state='m') which probably is the reason of some IDM deadlocks in the network.
I cannot find the message from @namdre anymore but remember that he advised not to use minor uncontrolled connections (state="m") at all. Instead, major uncontrolled connections and 'fake' additional traffic signals should be used. The network where this happens is attached.
[Tostmannplatz_VV.net.xml.zip](https://github.com/eclipse/sumo/files/2956609/Tostmannplatz_VV.net.xml.zip)

**Comment by namdre** [comment]
Generally speaking: what should be the default state when an uncontrolled connection is in conflict with a controlled connection? 
If there were vehicles on edge gneE55.0 when the traffic light enters a phase which allows traffic from east to south, there might be collisions with state 'M'.
I'm aware that this does not occur during the normal cycle (absent vehicles having a <stop> on that edge).

I think as a default behavior the current approach is ok (It's not a netedit issue per se. Netconvert -s old.net.xml -s new.net.xml would show the same change).
You can force the connection state to 'M' by setting pass=true for those connections and then netedit won't change the state either.

I would rephrase my earlier advice to: at traffic lights, either control the connections with (artifical) signals to define their state, or make sure they have state 'M'. If there is no conflict, they should get 'M' by default but if there is, you must manually override it because netconvert will always try to avoid unsafe junction rules.
